nengo_viz/static/viz_netgraph.js

```diff
@@ -8,9 +8,11 @@
  */
 VIZ.NetGraph = function(parent, args) {
     this.scale = 1.0;          // global scaling factor
-    this.offsetX = 0;          // global x,y pan offset 
+    this.offsetX = 0;          // global x,y pan offset
     this.offsetY = 0;
-    
+    this.zoom_fonts = false;    // scale fonts when zooming
+    this.font_size = 100;       // font size as a percent of base
```

Comment:
I would prefer to store the actual fraction, i.e. `font_size` divided by 100. Then it is not necessary to remember the scaling factor whenever this value is used (only for the input form it has to be converted to a percentage).